.github/workflows/ci.yml

```diff
@@ -40,6 +40,8 @@ jobs:
   tests:
     strategy:
       matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
(macos costs 10 times as much as linux, and windows twice as much as linux, so you've multiplied the usage of minutes of workalendar by 13. This impacts free minutes on the whole PeopleDoc organization. I'm not saying you have to change this, but be aware :) )

https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions

(yes, we need clear instructions on how we're supposed to use our minutes)